packs/ancestryfeatures/claws.json

```
@@ -45,6 +45,16 @@
                     "agile",
                     "unarmed"
                 ]
+            },
+            {
+                "itemType": "weapon",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:claw"
+                ],
+                "property": "other-tags",
+                "value": "lizardfolk-claws"
```

```suggestion
            },
            {
                "itemType": "weapon",
                "key": "ItemAlteration",
                "mode": "add",
                "predicate": [
                    "item:id:{item|id}"
                ],
                "priority": 119,
                "property": "other-tags",
                "value": "lizardfolk-claws"
```
Making sure we're targeting the right claw attack, also lowered the priority a bit to make sure it fires off before the Item Alterations in Iruxi Armaments